src/Api/Vault/AuthorizationHandlers/Collections/BulkCollectionAuthorizationHandler.cs

```diff
@@ -272,7 +272,7 @@ private async Task<bool> CanDeleteAsync(ICollection<Collection> resources, Curre
         if (_managedCollectionsIds == null)
         {
             var allUserCollections = await _collectionRepository
-                .GetManyByUserIdAsync(_currentContext.UserId!.Value, useFlexibleCollections: true);
+                .GetManyByUserIdAsync(_currentContext.UserId!.Value);
```

Comment:
There were a few places where we were passing in a constant value of `true`. I have made sure to already update the underlying sprocs/queries so that the new logic is always being executed. (e.g. I have copied the versioned sprocs back to their original non-versioned names.)